### DISCUSSION TITLE
History in the sidebar

### DISCUSSION THREAD

**Discussion by jokker23** [datalust/seq-tickets#1664] [Help] [answered]
Now that the history is in the side bar instead of the drop down I'm having difficulty with it as it truncates 80% of the line. 
Am I being thick? 
Is there a way to adjust the side bar size or view the history in a separate place?

Currently I'm hovering each line to see the entire text but it's very inefficient. 

Thanks!

**Comment by liammclennan** [comment, marked as answer]
You can provide custom css to allow the history items to wrap. 

In Settings -> INSTANCE add something like:

```css
.history > ul > li {
    white-space: normal;
    font-size: 12px;
}
```

![image](https://user-images.githubusercontent.com/271514/194178741-f5de5ba9-ed3c-4c41-9d11-b202e2923eeb.png)
